games-roguelike/cataclysm-dda/cataclysm-dda-0h.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils toolchain-funcs flag-o-matic
```

Comment:
Is there any reason you want to use xdg-utils.eclass?
With xdg.eclass, you don't need to define `pkg_postrm`. (You still need to call `xdg_pkg_postinst`, though.)